core/put/source/dev_ui.cpp

```diff
@@ -329,8 +329,8 @@ namespace put
             io.KeyMap[ImGuiKey_X] = PK_X;
             io.KeyMap[ImGuiKey_Y] = PK_Y;
             io.KeyMap[ImGuiKey_Z] = PK_Z;
-
-            io.RenderDrawListsFn = render;
+            
+            // io.RenderDrawListFn = render;
```

Comment:
if this is no longer needed, delete the whole line, no need keeping in redundant commented out code